maintainers/responsibilities.md

```diff
@@ -0,0 +1,44 @@
+# Maintainer responsibilities
+
+## Monitor notifications
+
+Watch discussions and proposals from these communication channels:
+
+- GitHub
+- Discourse
+- Matrix
+
+Maintain your entry in the [`CODEOWNERS`](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) file in the respective repository.
+GitHub will then request reviews for pull requests that change files you own.
+
+
+Set the notification level on the [Development > Documentation](https://discourse.nixos.org/c/dev/documentation/25) Discourse category to "watching".
```

Comment:
Since [providing guidance](https://github.com/NixOS/nix.dev/pull/362/files#diff-938b8f7d7edcc48d7885b5d53680b25ff11f0f3c8516e36acf51b7c7fc784c44R33) is also supposed to be part of the activities, probably watching the Help category is also required, although it has quite some traffic. But from experience, it's just scanning topics of interest and only reading and reacting to those where one can do something meaningful.